<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="1" ignored="0" total="6" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="40401" started-at="2018-02-14T13:34:54Z" finished-at="2018-02-14T13:35:34Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="40401" started-at="2018-02-14T13:34:54Z" finished-at="2018-02-14T13:35:34Z">
      <class name="Zmartasweden.Zmartasweden.Zmarta_se">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="beforeTest" is-config="true" duration-ms="13542" started-at="2018-02-14T19:04:54Z" finished-at="2018-02-14T19:05:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="VerifyPageTitle()[pri:0, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="VerifyPageTitle" duration-ms="118" started-at="2018-02-14T19:05:07Z" description="This Test Case is to Verify the Landing page of the Website" finished-at="2018-02-14T19:05:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPageTitle -->
        <test-method status="PASS" signature="VerifyLoan()[pri:1, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="VerifyLoan" duration-ms="14714" started-at="2018-02-14T19:05:07Z" description=" This Test Case  is to Verify the Applicant Loan Information" depends-on-methods="Zmartasweden.Zmartasweden.Zmarta_se.VerifyPageTitle" finished-at="2018-02-14T19:05:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLoan -->
        <test-method status="FAIL" signature="VerifyPersonalInfo()[pri:2, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="VerifyPersonalInfo" duration-ms="11925" started-at="2018-02-14T19:05:22Z" description=" This Test Case   is to Verify the Applicant Personal Information " depends-on-methods="Zmartasweden.Zmartasweden.Zmarta_se.VerifyLoan" finished-at="2018-02-14T19:05:34Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//div[@id='msg']/div/a"}  (Session info: chrome=64.0.3282.140)  (Driver info: chromedriver=2.34.522940 (1a76f96f66e3ca7b8e57d503b4dd3bccfba87af1),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 0 millisecondsFor documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.htmlBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:14.666Z'System info: host: 'LAPTOP-L45CF4AI', ip: '172.16.16.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'Driver info: org.openqa.selenium.chrome.ChromeDriverCapabilities {acceptSslCerts: true, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.34.522940 (1a76f96f66e3ca..., userDataDir: C:\Users\Jino\AppData\Local...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 64.0.3282.140, webStorageEnabled: true}Session ID: e48d85b0ec05f9ce5603ff6335d09025*** Element info: {Using=xpath, value=//div[@id='msg']/div/a}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@id='msg']/div/a"}  (Session info: chrome=64.0.3282.140)  (Driver info: chromedriver=2.34.522940 (1a76f96f66e3ca7b8e57d503b4dd3bccfba87af1),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 0 millisecondsFor documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.htmlBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:14.666Z'System info: host: 'LAPTOP-L45CF4AI', ip: '172.16.16.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'Driver info: org.openqa.selenium.chrome.ChromeDriverCapabilities {acceptSslCerts: true, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.34.522940 (1a76f96f66e3ca..., userDataDir: C:\Users\Jino\AppData\Local...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 64.0.3282.140, webStorageEnabled: true}Session ID: e48d85b0ec05f9ce5603ff6335d09025*** Element info: {Using=xpath, value=//div[@id='msg']/div/a}at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)at java.lang.reflect.Constructor.newInstance(Unknown Source)at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)at org.openqa.selenium.By$ByXPath.findElement(By.java:361)at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)at Pageobject.Locators.PersonalInfo(Locators.java:184)at Zmartasweden.Zmartasweden.Zmarta_se.VerifyPersonalInfo(Zmarta_se.java:98)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyPersonalInfo -->
        <test-method status="SKIP" signature="VerifyMaritalstatus()[pri:3, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="VerifyMaritalstatus" duration-ms="0" started-at="2018-02-14T19:05:34Z" description="  This Test Case is to Verify  the Applicant  Marital status " depends-on-methods="Zmartasweden.Zmartasweden.Zmarta_se.VerifyPersonalInfo" finished-at="2018-02-14T19:05:34Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Zmarta_se.VerifyMaritalstatus()[pri:3, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Zmarta_se.VerifyMaritalstatus()[pri:3, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8] depends on not successfully finished methodsat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:878)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyMaritalstatus -->
        <test-method status="SKIP" signature="VerifyEmploymentInfo()[pri:4, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="VerifyEmploymentInfo" duration-ms="0" started-at="2018-02-14T19:05:34Z" description="This Test Case is to Verify the Applicant Employment Information" depends-on-methods="Zmartasweden.Zmartasweden.Zmarta_se.VerifyMaritalstatus" finished-at="2018-02-14T19:05:34Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Zmarta_se.VerifyEmploymentInfo()[pri:4, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Zmarta_se.VerifyEmploymentInfo()[pri:4, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8] depends on not successfully finished methodsat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:878)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyEmploymentInfo -->
        <test-method status="SKIP" signature="VerifyFinalLoan()[pri:5, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="VerifyFinalLoan" duration-ms="0" started-at="2018-02-14T19:05:34Z" description="This Test Case is to  Verify  the Applicant Final Loan Information" depends-on-methods="Zmartasweden.Zmartasweden.Zmarta_se.VerifyEmploymentInfo" finished-at="2018-02-14T19:05:34Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Zmarta_se.VerifyFinalLoan()[pri:5, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Zmarta_se.VerifyFinalLoan()[pri:5, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8] depends on not successfully finished methodsat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:878)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyFinalLoan -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Zmartasweden.Zmartasweden.Zmarta_se@239963d8]" name="afterTest" is-config="true" duration-ms="1" started-at="2018-02-14T19:05:34Z" finished-at="2018-02-14T19:05:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- Zmartasweden.Zmartasweden.Zmarta_se -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
